portfolio/src/main/webapp/src/components/BookingForm.js

```diff
@@ -1,75 +1,100 @@
+            value={this.state.bookingDate}
+            onChange={this.handleChange}
+          />
+
+          <br />
+
+          <input
+            name="bookingDuration"
+            placeholder="Booking Duration (mins)"
+            type="number"
+            value={this.state.bookingDuration}
+            onChange={this.handleChange}
+          />
+
+          <br />
 
-    render() {
-        return (
-            <form>
-            <input
-                name="bookingName"
-                placeholder="Booking Name"
-                value={this.state.bookingName}
-                onChange={e => this.change(e)} 
-            />
-            
-            <br /> 
+          <input
+            name="bookingDescription"
+            placeholder="Booking Description"
+            value={this.state.bookingDescription}
+            onChange={this.handleChange}
+          />
 
-            <input
-                name="bookingDate"
-                placeholder="Booking Date"
-                value={this.state.bookingDate}
-                onChange={e => this.change(e)} 
-            />
-            
-            <br /> 
+          <br />
 
-            <input
-                name="bookingDuration"
-                placeholder="Booking Duration"
-                value={this.state.bookingDuration}
-                onChange={e => this.change(e)} 
-            />
-            
-            <br /> 
+          <input
+            name="bookingPrice"
+            placeholder="Booking Price ($)"
+            type="number"
+            value={this.state.bookingPrice}
+            onChange={this.handleChange}
+          />
 
-            <input
-                name="bookingDescription"
-                placeholder="Booking Description"
-                value={this.state.bookingDescription}
-                onChange={e => this.change(e)} 
-            />
-            
-            <br /> 
+          <br />
+          <Button
+            variant="contained"
+            color="secondary"
+            onChange={this.handleSubmit}
+          >
+             Book Now 
+          </Button>
+        </form>
 
-            <input
-                name="bookingPrice"
-                placeholder="Booking Price"
-                value={this.state.bookingPrice}
-                onChange={e => this.change(e)} 
-            />
-            
-            <br /> 
-                
-           <Button variant="contained" color="secondary" onClick={() => this.onSubmit()}> Book Now </Button>
-            </form>
-        )   
-    }
-}
\ No newline at end of file
+        <hr />
+        <h2> Entered information: </h2>
+        <p>Your Booking Name: {this.state.bookingName}</p>
+        <p>Your Booking Date: {this.state.bookingDate} </p>
+        <p>Your Booking Duration: {this.state.bookingDuration} (mins)</p>
+        <p>Your Booking Description: {this.state.bookingDescription} </p>
+        <p>Your Booking Price: ${this.state.bookingPrice} </p>
```

Comment:
does this need a $ sign?

Reply:
I kept it in order to style the price for the viewer!

Reply:
ah i see. nice!